.github/workflows/update-helm-chart.yml

```diff
@@ -0,0 +1,138 @@
+name: Upgrade Helm Chart
+on:
+  schedule:
+    - cron: "0 1 * * 1-5"
+  workflow_call:
+    inputs:
+      runtime_tag:
+        description: "Tag for the deno and python runner images"
+        type: "string"
+        required: true
+      scheduling-gates:
+        description: "Decides whether to use scheduling gates"
+        type: "string"
+        default: gates_off
+      allowed-namespaces:
+        description: "Decides whether to allow only certain namespaces"
+        type: "string"
+        default: allowed_ns_off
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  upgrade-helm-chart:
+    name: Upgrade Helm Chart
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        config:
+          - name: keptn
+            path: chart
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Install Helm
+        run: |
+          curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+          chmod 700 get_helm.sh
+          ./get_helm.sh
+
+      - name: Install Helm Chart
+        uses: ./.github/actions/install-helm-chart
+
+      - name: Upgrade Helm Chart
+        id: check-upgrades
+        run: |
+          export HELM_OLDER=$(helm list -A | grep keptn | awk '{print $9}')
+          HELM_UPGRADE_OUTPUT=$(helm upgrade keptn ./chart --install --namespace keptn --create-namespace)
+          export HELM_NEWER=$(helm list -A | grep keptn | awk '{print $9}')
```

Comment:
will this actually ever give you a newer version? You install the latest released version in the installation action, afterwards you install the local version which has the same version as the latest released one (until a new release is created -> then there is a version bump). Afterwards you have the same problem